selenium/src/test/java/com/epam/tc/hw2/ex1/Exercise1Test.java

```diff
@@ -0,0 +1,102 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.BaseChromeTest;
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Exercise1Test extends BaseChromeTest {
+
+    public SoftAssertions softly = new SoftAssertions();
+    private static final int ITEMS_IN_HEADER = 4;
+    private static final int NUMBER_OF_IMAGES = 4;
+    private static final int NUMBER_OF_ITEMS_IN_LEFT_SIDE_BAR = 5;
+
+    @Test(groups = {"HW2"})
+    public void exercise1() {
+
+        softly.assertThat(driver.getTitle()).as("Browser title is incorrect").isEqualTo("Home Page");
+
+        new WebDriverWait(driver, Duration.ofSeconds(5))
+                .until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("li.uui-profile-menu"))).click();
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+
+        /* Assert Username is loggined.*/
+        softly.assertThat(driver.findElement(By.id("user-name")).getText()).as("Roman Iovlev is not logged")
```

Comment:
I'd suggest moving `driver.findElement(...)` to a separate variable to improve the readability of code and make the element reusable